These tests are bound to fail dependending on QA tools updates

https://github.com/martinpitt/python-dbusmock/issues/28

--- a/tests/test_code.py	2017-08-16 09:20:43.848503995 +0200
+++ b/tests/test_code.py	2017-08-16 09:20:54.410177037 +0200
@@ -21,6 +21,7 @@
     pycodestyle = None
 
 
+@unittest.skip('style tests')
 class StaticCodeTests(unittest.TestCase):
     @unittest.skipIf(subprocess.call(['which', 'pyflakes'], stdout=subprocess.PIPE) != 0,
                      'pyflakes not installed')
